nimbus/src/nimbus.idl

```diff
@@ -20,12 +20,16 @@ dictionary EnrolledExperiment {
     string branch_slug;
 };
 
-dictionary ExperimentConfig {
+dictionary Config {
     string? server_url;
-    string? uuid;
     string? bucket_name;
 };
 
+dictionary AvailableRandomizationUnits {
+    string? client_id;
+    string? normandy_id;
```

Comment:
I don't expect we will *ever* support normandy id FWIW, it's a legacy thing from desktop.